web/srv/server.go

```diff
@@ -97,6 +97,7 @@ func NewServer(
 	server.router.GET("/namespaces/:namespace", handler.handleIndex)
 	server.router.GET("/daemonsets", handler.handleIndex)
 	server.router.GET("/statefulsets", handler.handleIndex)
+	server.router.GET("/jobs", handler.handleIndex)
```

Comment:
a few lines below here, add:
```go
server.router.GET("/namespaces/:namespace/jobs/:job", handler.handleIndex)
```

(you can verify what this fixes by hard-refreshing a `jobs` page on the dashboard)...

![screen shot 2019-03-04 at 2 37 42 pm](https://user-images.githubusercontent.com/236915/53767588-1d08c100-3e8b-11e9-9997-6457b1347ea1.png)

Reply:
Done!